src/transformers/trainer.py

```diff
@@ -471,6 +471,10 @@ def __init__(
             if self.args.fsdp_config.get("limit_all_gathers", False):
                 self.limit_all_gathers = True
 
+            self.activation_checkpointing = False
+            if self.args.fsdp_config.get("activation_checkpointing", False):
+                self.activation_checkpointing = True
+
```

Comment:
not required, pleasec remove these lines

Reply:
Done.